dbdialog/src/main/java/org/pentaho/ui/database/event/DbInfoHandler.java

```diff
@@ -0,0 +1,15 @@
+package org.pentaho.ui.database.event;
+
+import org.pentaho.di.core.database.DatabaseMeta;
+
+/**
+ * A complement to {@link DataHandler} that handles
+ * database meta save/load without having to keep
+ * adding subtypes to a single class.
+ */
+public interface DbInfoHandler {
+  /** ui -> meta */
+  void getConnectionSpecificInfo( DatabaseMeta meta );
+  /** meta -> ui */
+  void setConnectionSpecificInfo( DatabaseMeta meta );
```

Comment:
get/set seem like the wrong verbs here.

Reply:
was following the methods they're called in, but probably better to change to load/save